test/fixtures/rules/std_rule_cases/L057.yml

```diff
@@ -0,0 +1,81 @@
+rule: L057
+
+test_fail_special_chars_create_table_space:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "Internal Space" INT
+    )
+
+test_fail_special_chars_create_table_gt:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "Greater>Than" INT
+    )
+
+test_fail_special_chars_create_table_lt:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "Less<Than" INT
+    )
+
+test_fail_special_chars_create_table_hash_unquoted:
+  fail_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        Number# INT
+    )
+  configs:
+    core:
+      dialect: tsql
+
+test_pass_special_chars_create_table_none:
+  pass_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "No_Internal_Space" INT,
+        GreaterThan INT,
+        LessThan INT,
+        NumberVal INT
+    )
+
+test_pass_special_chars_create_table_none_quoted_tsql:
+  pass_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        [No_Internal_Space] INT,
+        [GreaterThan] INT,
+        [LessThan] INT,
+        NumberVal INT
+    )
+  configs:
+    core:
+      dialect: tsql
+
+
+test_pass_special_chars_create_table_space_allowed:
+  pass_str:
+    CREATE TABLE DBO.ColumnNames
+    (
+        "Internal Space" INT
+    )
+  configs:
+    rules:
+      L057:
+        allow_space_in_identifier: True
+
+test_fail_special_chars_alias:
+  fail_str:
+    SELECT a as "alias with space"
+    FROM DBO.ColumnNames
+
+test_pass_special_chars_reference:
+  pass_str:
+    SELECT "column with space" as a
+    FROM DBO.ColumnNames
+  configs:
+    rules:
+      L057:
+        quoted_identifiers_policy: aliases
```

Comment:
maybe include some test cases with custom quoted_identifiers_policy configs and also some where one of the identifiers policies are set to none to verify they pass as expected

Reply:
@jpy-git test cases have been expanded!